src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const newObj = {
+    result: 0,
+
+    reset() {
+      newObj.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
+      callback(value);
+
+      return this;
+    },
+
+    add(value) {
+      newObj.result = newObj.result + value;
```

Comment:
Instead of 'newObj.result = newObj.result + value;', you can simply use 'this.result += value;'.